package.json

```diff
@@ -25,10 +30,32 @@
     "@vitejs/plugin-vue": "^3.2.0",
     "@vue/test-utils": "^2.2.4",
     "autoprefixer": "^10.4.13",
+    "eslint": "^7.32.0",
+    "eslint-plugin-vue": "^8.0.3",
     "jsdom": "^20.0.3",
     "postcss": "^8.4.19",
     "tailwindcss": "^3.2.4",
-    "vite": "^3.2.4",
+    "vite": "^3.2.5",
     "vitest": "^0.25.3"
-  }
+  },
+  "eslintConfig": {
+    "root": true,
+    "env": {
+      "node": true
+    },
+    "extends": [
+      "plugin:vue/vue3-essential",
+      "eslint:recommended"
+    ],
+    "parserOptions": {
+      "parser": "@babel/eslint-parser"
+    },
+    "rules": {}
+  },
+  "browserslist": [
+    "> 1%",
+    "last 2 versions",
+    "not dead",
+    "not ie 11"
+  ]
```

Comment:
It's better to have a separate config file.

Reply:
browserslist: has been removed also.